articles/libraries/auth0js/v9/migration-v7-v9.md

```diff
@@ -8,182 +8,170 @@ toc: true
 
 This guide includes all the information you need to update [Auth0.js](/libraries/auth0js) from v7 to v9. Find out if you should upgrade or not by reading [Migrating to Auth0.js v9](/libraries/auth0js/v9/migration-guide).
 
+The [Auth0.js v7 to v8 Migration Guide](/libraries/auth0js/v8/migration-guide) has detailed information on how to migrate from v7 to v8, and that information is still valid, v7 is very similar to v8. This document will go over some common usage patterns, and focus on things that changed after that guide was created.
+
 ## Migration Steps
 
 <%= include('../../_includes/_get_auth0_js_latest_version') %>
 
-### Follow the Auth0.js v7 to v8 Migration Guide
-
-Follow the [instructions to migrate from Auth0.js v7 to v8](/libraries/auth0js/v8/migration-guide).
-
-<%= include('../../_includes/_configure_embedded_login') %>
-<%= include('../../_includes/_change_get_profile') %>
-<%= include('../../_includes/_review_get_ssodata') %>
-
-## Behavioral Changes
-
-<%= include('../../_includes/_hosted_pages') %>
-<%= include('../../_includes/_default_values') %>
-
-## Using Auth0.js to log in users
+## Using Auth0.js to Log In Users
 
 ### Using Auth0.js v7
 
 ```js
 var auth0 = new Auth0({
-  domain: "${account.namespace}",
-  clientID: "${account.clientId}",
-  responseType: "token"
+  domain:'${account.namespace}'),,
```

Comment:
Hey @aaguiarz the `),` characters (in the `domain:'${account.namespace}'),,`) look like a typo